client/src/components/Account/AccountDropdown.tsx

```diff
@@ -154,13 +154,16 @@ export const AccountDropdown = forwardRef<HTMLDivElement, AccountDropdownProps>(
               </NavDropdownItem>
             </ItemAndIconContainer>
           )}
-          <ItemAndIconContainer onClick={handleReceiveClick}>
-            <StyledArrowDownCircle />
 
-            <NavDropdownItem>
-              Receive
-            </NavDropdownItem>
-          </ItemAndIconContainer>
+          {loginStatus != LoginStatus.EOA && (
```

Comment:
We should keep this since we will be using it for any off ramper who has insufficient balance. There should be some way for them to navigate to this modal outside of that flow